classes/stock/StockMvt.php

```diff
@@ -132,7 +132,7 @@ class StockMvtCore extends ObjectModel
             'employee_firstname' => ['type' => self::TYPE_STRING, 'validate' => 'isName'],
             'employee_lastname' => ['type' => self::TYPE_STRING, 'validate' => 'isName'],
             'id_stock' => ['type' => self::TYPE_INT, 'validate' => 'isUnsignedId', 'required' => true],
-            'physical_quantity' => ['type' => self::TYPE_INT, 'validate' => 'isUnsignedInt', 'required' => true],
+            'physical_quantity' => ['type' => self::TYPE_INT,  'validate' => 'isUnsignedId', 'required' => true],
```

Comment:
isUnsignedId?
physical_quantity doesn't seem to be an id field, so don't think it should use id-specific validation.

Reply:
because it can't be - only positive number. By doing this we increase the max size 2x.

Reply:
yeah I get it (probably not worth changing anything here), just wanted to point out that the whole concept seems wrong (it just happens to work.. :smile:).
The naming suggests its validation for the `id` so it means the validation itself could in theory be changed to suit all the id's at some point, but won't necessary work for the physical quantities anymore.
Ofc it is legacy code and probably won't ever change, so lets say its just the method name that is incorrect... Im ok with it as long as @jolelievre is ok with it :smile:

Reply:
You're right @zuk3975 especially since we have a `isUnsignedInt` validator as well, so it would probably be more appropriate

Reply:
Tldr: just use `isUnsignedInt` here.
Also there is one unnecessary space:
```suggestion
            'physical_quantity' => ['type' => self::TYPE_INT, 'validate' => 'isUnsignedInt', 'required' => true],
```